base/applications/calc/winmain.c

```diff
@@ -1056,9 +1066,8 @@ static char *handle_sequence_input(HWND hwnd, sequence_t *seq)
         }
     } else
     if (ch == '$') {
-        calc.ptr =
-        _tcscpy(calc.buffer, calc.source) +
-        _tcslen(calc.source);
+        StringCbCopy(calc.buffer, sizeof(calc.buffer), calc.source);
+        calc.ptr = calc.buffer + _tcslen(calc.source);
```

Comment:
StringCbCopyEx may help simplifying this.

Reply:
Fixed.